posthog/test/test_feature_flag.py

```diff
@@ -76,7 +77,7 @@ def test_multiple_flags(self):
                         ],
                         "rollout_percentage": 100,
                     },
-                    {"rollout_percentage": 50},
+                    {"rollout_percentage": 100},
```

Comment:
whoops, ignore this, not important, I was confused for a sec why I was losing "one" in the result